components/email/src/Email.svelte

```diff
@@ -645,6 +647,33 @@
       participants[0].email !== messages[messages.length - 1]?.from[0].email
     );
   }
+
+  type FormatParticipantOptions = {
+    format?: "full";
+    self?: boolean;
+  };
+
+  function formatParticipant(
+    participant: Participant,
+    options: FormatParticipantOptions = { format: "full", self: false },
+  ) {
+    let { format, self } = options;
+    let participantString: string;
+
+    switch (format) {
```

Comment:
Can we do away with `format` as a param for now? We can re-add it once we have a need for multiple display formats, but for now this is a bit overkill.